datastore/db/metadata/inspection/__init__.py

```diff
@@ -114,13 +128,179 @@ class Inspection(BaseModel):
     product: ProductInformation
     cautions: SubLabel
     instructions: SubLabel
-    micronutrients: ValuesObjects
-    ingredients: ValuesObjects
+    micronutrients: LocalizedNamedValue
+    ingredients: LocalizedNamedValue
     specifications: Specifications
     first_aid: SubLabel
     guaranteed_analysis: List[Value]
 
 
+# Temporary
+class InspectionV2(BaseModel):
```

Comment:
Why are we naming objects V2? This is quite problematic

Reply:
This was to avoid breaking tests. Hence the `#Temporary` comment.